app/frontend/src/features/communities/events/LongEventCard.tsx

```diff
@@ -0,0 +1,203 @@
+    width: "2.5rem",
+  },
+  extraAvatarCount: {
+    marginInlineStart: theme.spacing(1),
+  },
+}));
+
+export const AVATAR_GROUP_TEST_ID = "avatar-group";
+
+export interface LongEventCardProps {
+  event: Event.AsObject;
+}
+
+export default function LongEventCard({ event }: LongEventCardProps) {
+  const classes = useStyles();
+  const theme = useTheme();
+  const isBelowLg = useMediaQuery(theme.breakpoints.down("md"));
+
+  const truncatedContent = useMemo(
+    () =>
+      getContentSummary({
+        originalContent: event.content,
+        maxLength: isBelowLg ? 120 : 300,
+      }),
+    [event.content, isBelowLg]
+  );
+  const startTime = dayjs(timestamp2Date(event.startTime!));
+  const endTime = dayjs(timestamp2Date(event.endTime!));
+  const isSameDay = endTime.isSame(startTime, "day");
+
+  const { attendees, attendeesIds, isAttendeesLoading } = useEventAttendees({
+    eventId: event.eventId,
+    type: "summary",
+  });
+  const extraAvatarCount = event.goingCount - Math.min(3, attendeesIds.length);
+
+  return (
+    <Card className={classes.overviewRoot}>
+      <Link
+        className={classes.overviewContent}
+        to={routeToEvent(event.eventId, event.slug)}
+      >
+        <div className={classes.eventInfoContainer}>
+          <Typography variant="h2">{event.title}</Typography>
+          <div className={classes.eventTimeContainer}>
+            <Typography className={classes.eventTime} variant="body1">
+              {startTime.format("ll")}
+            </Typography>
+            <Typography className={classes.eventTime} variant="body1">
+              {`${startTime.format("LT")} ${TO} ${endTime.format(
+                isSameDay ? "LT" : "lll"
+              )}`}
+            </Typography>
+          </div>
+        </div>
+        <Typography className={classes.content} variant="body1">
+          {truncatedContent}
+        </Typography>
+        <img
+          alt=""
+          className={classes.image}
+          src={event.photoUrl || eventImagePlaceholder}
+        />
+        <div className={classes.attendeesAndInfo}>
+          <Typography
+            className={classes.onlineOrOfflineInfo}
+            color="primary"
+            variant="body1"
+          >
+            {event.offlineInformation
+              ? event.offlineInformation.address
+              : ONLINE}
+          </Typography>
+          <Typography variant="body1">
+            {getAttendeesCount(event.goingCount)}
+          </Typography>
+          {isAttendeesLoading ? (
+            <CircularProgress />
+          ) : (
+            attendees &&
+            attendeesIds.length > 0 && (
+              <div
+                className={classes.avatarGroup}
+                data-testid={AVATAR_GROUP_TEST_ID}
+              >
+                {attendeesIds.slice(0, 3).map((attendeeUserId) => {
+                  const attendee = attendees.get(attendeeUserId);
+                  return (
+                    <Avatar
```

Comment:
Next time, there is an `AvatarGroup` mui component

Reply:
I used that to start with, but they hardcode the circle so that for events with a lot of attendees (like +100), the text gets clipped off. Also had to return "dummy avatars" JSX just so the components get the count correctly so I thought it was easier to write my own here in the end 😅